src/backend/distributed/commands/table.c

```diff
@@ -1728,8 +1747,15 @@ PostprocessAlterTableStmt(AlterTableStmt *alterTableStatement)
 								if (ShouldSyncTableMetadata(relationId) &&
 									ClusterHasKnownMetadataWorkers())
 								{
+									newSequencesMetadataSync = true;
 									MarkSequenceDistributedAndPropagateDependencies(
 										seqOid);
+
+									AppendAlterTableWithNextvalDefaultStmt(
+										&alterTableDefaultNextvalStmt, seqOid,
```

Comment:
could AppendAlterTableWithNextvalDefaultStmt just return a char *? The use of StringInfo seems redundant.